shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -463,9 +604,9 @@ - (NSRect)firstRectForCharacterRange:(NSRange)range actualRange:(NSRangePointer)
         CGRectApplyAffineTransform(_caretRect, CATransform3DGetAffineTransform(_editableTransform));
 
     // flip and convert to screen coordinates
-    double viewHeight = self.flutterViewController.view.bounds.size.height;
+    double viewHeight = self.flutterViewController.flutterView.bounds.size.height;
```

Comment:
We should avoid forcing the view to load here. If view is not loaded, We should just return `CGRectZero`.

Reply:
Alternatively, we should just access the `window` property from the view controller and use that if appropriate.